src/app/common/stacks-utils.ts

```diff
@@ -76,3 +77,80 @@ export function getSafeImageCanonicalUri(imageCanonicalUri: string, name: string
     ? imageCanonicalUri
     : '';
 }
+
+/**
```

Comment:
These should all be relocated soon with the `@leather-wallet/stacks` work, but wanted to move them with other stacks specific utils for now. Let me know if that seems incorrect, but I'm not sure why they would be in the `ui` lib?

Reply:
yeah, ui lib doesn't seems as correct place for them
may be refactor it to fn declarations while here?

Reply:
I'll do this!